static_src/components/base_component.jsx

```diff
@@ -0,0 +1,8 @@
+
+import React from 'react';
+
+export default class BaseComponent extends React.Component {
```

Comment:
What's the purpose of this since all of the other components extend `React.Component`? (E.g. Just add a comment if necessary)